cmd/ssl-handshake-check/main.go

```diff
@@ -15,17 +15,52 @@
 package main
 
 import (
+	"context"
 	"os"
+	"strconv"
+	"time"
 
 	"github.com/Comcast/kuberhealthy/v2/pkg/checks/external/checkclient"
+	"github.com/Comcast/kuberhealthy/v2/pkg/checks/external/nodeCheck"
 	"github.com/Comcast/kuberhealthy/v2/pkg/checks/external/ssl_util"
+	"github.com/Comcast/kuberhealthy/v2/pkg/kubeClient"
 	log "github.com/sirupsen/logrus"
+	"k8s.io/client-go/kubernetes"
 )
 
-var domainName string
-var portNum string
+const defaultCheckTimeout = 10 * time.Minute
+
+var (
+	kubeConfigFile = os.Getenv("KUBECONFIG")
+	checkTimeout   time.Duration
+	domainName     string
+	portNum        string
+	selfSigned     string
+	selfSignedBool bool
+	ctx            context.Context
+)
+
+type Checker struct {
+	client         *kubernetes.Clientset
+	domainName     string
+	portNum        string
+	selfSignedBool bool
+}
 
 func init() {
+	var err error
```

Comment:
is this needed? seems like its instantiated in line 57

Reply:
Updated.